pennylane_qiskit/converter.py

```diff
@@ -91,15 +154,38 @@ def _check_circuit_and_bind_parameters(
             "The circuit {} is not a valid Qiskit QuantumCircuit.".format(quantum_circuit)
         )
 
+    # confirm parameter names are valid for conversion to PennyLane
+    for name in ["wires", "params"]:
+        if name in [p.name for p in quantum_circuit.parameters]:
+            raise RuntimeError(
```

Comment:
No, it's not mentioned anywhere currently. This will only come up if users do something horrible like:

```
p = Parameter("wires")

qc = QuantumCircuit(2, 2)
qc.rx(p, 0)
qc.rx(p, 1)

qml.from_qiskit(qc)(wires=0.23)
```

The docstring of `_function` below definitely needs to be updated to reflect the new UI, and that would be a reasonable place to note it.